src/panel/spoiler_card/SpoilerCardList.js

```diff
@@ -11,7 +11,7 @@ const SpoilerCardList = observer(props =>
 				<SpoilerCardContainer
 					index={index}
 					title={obj["title"]}
-					tags={obj["tags"].join(", ")}
```

Comment:
The OR operator can map back to `|` or `,`. Adding/removing brackets might also be confusing.